csunplugged/topics/models.py

```diff
@@ -233,6 +233,21 @@ def __str__(self):
         )
 
 
+class AgeRange(models.Model):
+    """Model for age range in database."""
+
+    #  Auto-incrementing 'id' field is automatically set by Django
+    age_range = IntegerRangeField()
+
+    def __str__(self):
+        """Text representation of AgeRange object.
+
+        Returns:
+            Integer range (str).
+        """
+        return repr(self.age_range)
```

Comment:
it is, not having it just results in "AgeRange object", not having the ``repr`` results in an error (because tuple not string)